src/framing/FrameSerializer_v1_0.cpp

```diff
@@ -65,6 +66,7 @@ static void serializeMetadataInto(
 
   // Use signed int because the first bit in metadata length is reserved.
   if (metadata->length() > kMaxMetadataLength) {
+    LOG(ERROR) << "Metadata is too big to serialize.";
     CHECK(false) << "Metadata is too big to serialize";
```

Comment:
the CHECK here will already log (even though it should really be `LOG(FATAL)` instead).